src/jsMain/kotlin/baaahs/app/settings/UiSettings.kt

```diff
@@ -6,6 +6,7 @@ import kotlinx.serialization.Serializable
 @Serializable
 data class UiSettings(
     val darkMode: Boolean = true,
+    val autoMode: Boolean = false,
```

Comment:
I think `autoMode` makes more sense as show state rather than client config state. `UiSettings` is meant for config on the local client, and doesn't get synched to other open clients, but you probably want `autoMode` to show the same value on every open client. Otherwise there'd be no way to stop an unattended iPad from monkeying with your show.

Thinking about it a little more, implementing it that way actually raises some weird issues: it implies that some single entity needs to be in charge of the monkeying, which makes me think it really should be happening on the server. Further, if there's no client currently connected I think we'd still like monkeying to happen potentially, another vote in favor of the server doing it.

... which would require rethinking our approach a little bit. Maybe it makes sense to get it working as-is since time grows short. 🤷‍♀️